src/EspMQTTClient.cpp

```diff
@@ -542,9 +542,9 @@ void EspMQTTClient::mqttMessageReceivedCallback(char* topic, byte* payload, unsi
     if (mqttTopicMatch(mTopicSubscriptionList[i].topic, String(topic)))
     {
       if(mTopicSubscriptionList[i].callback != NULL)
-        (*mTopicSubscriptionList[i].callback)(payloadStr); // Call the callback
+        (mTopicSubscriptionList[i].callback)(payloadStr); // Call the callback
```

Comment:
The parentheses are not required anymore  (mTopicSubscriptionList[i].callback)(payloadStr); ->  mTopicSubscriptionList[i].callback(payloadStr);